apps/frontend/src/components/card/MapPreview.tsx

```diff
@@ -20,15 +21,15 @@ const MapPreviewContainer = styled.div`
   background-image: url(${backgroundMapImage});
 `;
 
-const MapPreview = ({ draft, handleRemove, handleCopy }: MapPreviewInterface) => {
+const MapPreview = ({ draft, handleRemove, handleCopy, disable }: MapPreviewInterface) => {
```

Comment:
You should set a default, for boolean it's easy :
```suggestion
const MapPreview = ({ draft, handleRemove, handleCopy, copyEnabled = false }: MapPreviewInterface) => {
```

> https://betterprogramming.pub/5-recipes-for-setting-default-props-in-react-typescript-b52d8b6a842c

Reply:
I understand better how to properly implement this now. Thanks